### PR TITLE
feat: display example MVP (DCP-1598)

### PR DESCRIPTION
**Pull request by jimmymadrigal**
This is the first example that simply shows the User data and the Content owner username if present
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "frames-examples-pnpm",
```
Comment (vinayakkulkarni) [inline package.json]:
```suggestion
  "name": "@dscvr-one/frames-examples",
```

Ref: https://dscvr.atlassian.net/browse/DCP-1591

File: examples/framejs-nextjs-display-info/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "framejs-nextjs-display-info",
```
Comment (ncpenke) [inline examples/framejs-nextjs-display-info/package.json]: Wdyt of calling this example `framesjs-nextjs-display-user-info`?

File: examples/framejs-nextjs-display-info/app/page.tsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+          </FrameImage>
+          <FrameButton>Refresh</FrameButton>
+        </FrameContainer>
+      );
+    }
+  } else if (previousFrame.postBody) {
+    const frameMessage = await getFrameMessage(previousFrame.postBody, {
+      fetchHubContext: false,
+    });
+    console.log('frameMessage', frameMessage);
```
Comment (PipDscvr) [inline examples/framejs-nextjs-display-info/app/page.tsx]: do you need this console.log ?

File: examples/framejs-nextjs-display-info/tailwind.config.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import type { Config } from "tailwindcss";
+
+const config: Config = {
+  content: [
+    "./pages/**/*.{js,ts,jsx,tsx,mdx}",
```
Comment (PipDscvr) [inline examples/framejs-nextjs-display-info/tailwind.config.ts]: I dont see the usage of pages or components, I wonder if we should keep those lines or not?

File: package.json (hunk 1 above)
Comment (jimmymadrigal) [reply package.json]: Thanks done

File: examples/framejs-nextjs-display-info/package.json (hunk 2 above)
Comment (jimmymadrigal) [reply examples/framejs-nextjs-display-info/package.json]: perfect, sounds better. done

File: examples/framejs-nextjs-display-info/app/page.tsx (hunk 3 above)
Comment (jimmymadrigal) [reply examples/framejs-nextjs-display-info/app/page.tsx]: thanks, removed

File: examples/framejs-nextjs-display-info/tailwind.config.ts (hunk 4 above)
Comment (jimmymadrigal) [reply examples/framejs-nextjs-display-info/tailwind.config.ts]: right, removed

**Review by PipDscvr — APPROVED** [review]
LGTM. Thanks

**Review by ncpenke — APPROVED** [review]

**Pull request closed without merging**